### ISSUE TITLE
Deepcell predicts huge and empty cells in data

### ISSUE THREAD

**Issue by YubinXie** [vanvalenlab/deepcell-tf#465]
**Describe the bug**
Deepcell is amazing when the image is full of cells. But where there are empty space, there will be empty (no nuclei and membrane) and huge (super big cell) prediction. 

**To Reproduce**
Multiplexed segmentation with the attached image.

**Screenshots**

![99A7DAA6-CD4A-4D3A-B76F-BA564CAE7FF1](https://user-images.githubusercontent.com/16257776/101220418-a26a6800-3653-11eb-867a-b9047886f479.png)

**Comment by ngreenwald** [comment]
Can you post the code you're using to generate the output and attach the image file?

**Comment by YubinXie** [comment]
Hi Noah, thanks for the quick response!
```
from deepcell.applications import NuclearSegmentation, MultiplexSegmentation, CytoplasmSegmentation
appm = MultiplexSegmentation()
im=np.load('multiplex_seg.npy')
labeled_image = appm.predict(im)

```
[multiplex_seg.npy.zip](https://github.com/vanvalenlab/deepcell-tf/files/5645834/multiplex_seg.npy.zip)

**Comment by ngreenwald** [comment]
What version of DeepCell-tf are you using? I don't see any issues running the latest model using the cloud server: 
![image](https://user-images.githubusercontent.com/13770365/101222069-d76bc000-363d-11eb-991b-66806a80a45a.png)

**Comment by YubinXie** [comment]
I am using 0.8.1 with the multiplexed function (two channels)

**Comment by YubinXie** [comment]
The prediction is normal when I only feed the nuclear channel.

**Comment by ngreenwald** [comment]
What are the row/column coordinates of the area where you're seeing that in the original image?

**Comment by YubinXie** [comment]
I am uploading the predicted image with X, Y-axis info. The results might be slightly different from the screenshot before but it is able to reproduce the problem. 

![segmentation](https://user-images.githubusercontent.com/16257776/101439351-54b75f00-38e2-11eb-83eb-85e3d3e8d401.png)

**Comment by ngreenwald** [comment]
Hi Yubin, I don't see those when I run it on my end. Can you make sure you've updated your docker image and are using the latest version of the repo?

**Comment by YubinXie** [comment]
Hi Noah, I just downloaded and tested with the latest repo here v0.8.2. It still has the same problem. Do you think it could be the hyper-parameters/post-preprocessing/model weights difference from the GitHub one and the API online one?

**Comment by YubinXie** [comment]
for example, i saw 'Rescale Automatically' option in online website but I dont see any preprocessing options in the github repo version.

**Comment by ngreenwald** [comment]
Very strange. After updating everything, now I'm seeing the same problem. Before I wasn't seeing this issue even when I did it locally using the deepcell-tf repo. I'll look into it, the web server and the github are using the same pretrained weights files, not sure why we're only seeing these errors in one.

**Comment by ngreenwald** [comment]
@willgraf It looks to me like the consumer is using the 5th iteration of the multiplex model, whereas the application is using the bundled version from the 6th. I don't think we ever switched the consumer over to the TF2 trained version, which appears to be what is causing the problems. All of the commits following TF2 migration have models which produce this error, whereas the one immediately preceding the migration does not. It's either an issue with this specific model that I trained, or there's a broader issue with normalization in the TF2 build. 

Is it possible to create the necessary weights file for the application based on the "5" folder that the consumer is loading from?

**Comment by ngreenwald** [comment]
@YubinXie the commit immediately prior to the TF2 migration (https://github.com/vanvalenlab/deepcell-tf/commit/39fab86326cb3d2bce685c860b7e11377a73dc03) has the version of the model which doesn't produce these errors

**Comment by willgraf** [comment]
Yes, it does look like the consumer is using the multiplex model v5 while deepcell v8+ is using multiplex model v6. AFAIK, it is not possible to take an exported model from TF1 and load that model into TF2, and re-training the model in TF2 is required.

**Comment by YubinXie** [comment]
@willgraf @ngreenwald Thanks for working on this guys! What would be the best way for me to update the weight? Update the deep cell or?

**Comment by ngreenwald** [comment]
Yeah, if you pull from master and reload the application, you'll have the updated weights. It should be version 7, it will tell you when it's downloading the weights.

**Comment by YubinXie** [comment]
@ngreenwald Got it. Thanks. Any chance version 8.x will be updated with correct and new weights?

**Comment by ngreenwald** [comment]
Yup, patch coming within a week or so
